README.md

```diff
@@ -1,2 +1,64 @@
-# MC-Parallel-Functions
-Parallel Functions Morning Challenge
+**Author**: [@des-des](https://github.com/des-des)  
+
+**Maintainer**: TBC
```

Comment:
I think we can remove the "maintainer" header, it's no longer used for anything.